config/locales/views/graders/en.yml

```diff
@@ -18,9 +18,12 @@ en:
     help: Assign graders to specific student groups and criteria. By default, graders will be assigned to all criteria.
     hide_unassigned_criteria: Only show assigned criteria to graders (if unselected, all graders will be able to see all criteria)
     no_grader_assigned: 'Default grader: course instructor(s)'
+    random_instruction: Assign a weight for randomized distribution
```

Comment:
Okay, please reword to "Assign a weight for each grader for randomized assignment. With the default weights of 1, all graders will be assigned an equal number of groups." Make sure to include the punctuation.